Gemfile

```diff
@@ -1,8 +1,10 @@
 source "https://rubygems.org"
 
 group :development do
-  gem "test-kitchen", :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
-  gem 'kitchen-vagrant', git: 'https://github.com/test-kitchen/kitchen-vagrant.git', :branch => 'windows-guest-support'
+  gem "test-kitchen"
+  gem 'kitchen-pester'
+  gem 'kitchen-vagrant'
+  gem 'vagrant-wrapper', '>= 2.0'
```

Comment:
@ksubrama I'm not actually sure.  I've been testing locally with my hyperv driver since it is orders of magnitude faster.